gui/wxpython/location_wizard/wizard.py

```diff
@@ -100,7 +100,11 @@ def __init__(self, parent, title):
 
         # page title
         self.title = StaticText(
-            parent=self, id=wx.ID_ANY, label=title, style=wx.ALIGN_CENTRE_HORIZONTAL
+            parent=self,
+            id=wx.ID_ANY,
+            label=title,
+            style=wx.ALIGN_CENTRE_HORIZONTAL,
+            size=(-1, 30),
```

Comment:
Is there a way to get the actual font height here instead of hard-coding 30 pixels? Doing so would be more flexible and safer.